libs/designsystem/src/lib/components/tabs/tabs.component.spec.ts

```diff
@@ -17,7 +17,37 @@ describe('TabsComponent', () => {
     spectator = createHost({});
   });
 
-  it('should create', () => {
-    expect(spectator.component).toBeTruthy();
+  describe('hideTabBar: true', () => {
+    it('should hide tab bar', () => {
+      spectator.component.tabBarBottomHidden = true;
+      spectator.detectChanges();
+      const ionTabBarElm = spectator.query('ion-tab-bar');
+
+      expect(ionTabBarElm).toHaveComputedStyle({ display: 'none' });
+    });
+    it('should NOT set page footer safe area override if tab bar bottom is hidden', () => {
+      spectator.component.tabBarBottomHidden = true;
+      spectator.detectChanges();
+      const ionTabBarElm = spectator.query('ion-tab-bar');
+
+      expect(ionTabBarElm).toHaveComputedStyle({ '--kirby-page-footer-safe-area-bottom': '' });
+    });
+  });
+
+  describe('hideTabBar: false', () => {
+    it('should show tab bar', () => {
+      spectator.component.tabBarBottomHidden = false;
+      spectator.detectChanges();
+
+      const ionTabBarElm = spectator.query('ion-tab-bar');
+      expect(ionTabBarElm).not.toHaveComputedStyle({ display: 'none' });
+    });
+    it('should set footer safe area to 0 when tab bar is shown and in bottom', () => {
```

Comment:
Please add test for window size <large (use `TestHelper.resizeTestWindow` + `TestHelper.screensize` - see `modal.helper.spec.ts` for reference)